aws/data_source_aws_outposts_outpost_instance_type_test.go

```diff
@@ -0,0 +1,75 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccAWSOutpostsOutpostInstanceTypeDataSource_InstanceType(t *testing.T) {
+	dataSourceName := "data.aws_outposts_outpost_instance_type.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSOutpostsOutposts(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSOutpostsOutpostInstanceTypeDataSourceConfigInstanceType(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestMatchResourceAttr(dataSourceName, "instance_type", regexp.MustCompile(`^.+$`)),
```

Comment:
`testAccCheckResourceAttrNonEmptyString()` would be a useful check

Reply:
Yes it would! 😉